pkg/activator/net/throttler.go

```diff
@@ -305,16 +313,7 @@ func (rt *revisionThrottler) updateCapacity(backendCount int) {
 		return len(assigned)
 	}()
 
-	capacity := 0
-	if numTrackers > 0 {
-		// Capacity is computed based off of number of trackers,
-		// when using pod direct routing.
-		capacity = rt.calculateCapacity(len(rt.podTrackers), ac)
```

Comment:
I wonder why this wasn't using assignedTrackers before

Reply:
I am not sure.
I found the PR adding calculateCapacity function, but I couldn't find any reason why used podTrackers instead of assignedTrackers
https://github.com/knative/serving/pull/5008